atlante/server/coordinator/postgresql/postgresql.go

```diff
@@ -501,6 +505,119 @@ ORDER BY jobstatus.id desc limit 1;
 	}, true
 }
 
+func genAllSQL(primarySQL, defaultSQL string, limit uint) (string, error) {
```

Comment:
can we get a comment on this func. The name and associated code are not clear to me.